static/schemas/neon-api/v1/neon_gasPrice/response.json

```diff
@@ -0,0 +1,141 @@
+{
+  "type": "object",
+  "description": "Response to request for transaction gas price",
+  "required": [
+    "jsonrpc",
+    "id",
+    "result"
+  ],
+  "properties": {
+    "jsonrpc": {
+      "type": "string",
+      "default": "2.0",
+      "description": "The JSON RPC version"
+    },
+    "id": {
+      "type": "string",
+      "examples": [
+        "OpEgqw7r98qwerqwehkjhwe8"
+      ],
+      "description": "End-user provided identifier for the request"
+    },
+    "result": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "gas_price": {
+            "description": "Gas fee as hexadecimal value in Galan (NEON).",
+            "type": "string",
+            "format": "hexadecimal",
+            "examples": [
+              "0x0"
+            ]
+          },
+          "suggested_gas_price": {
+            "description": "Suggested gas price as hexadecimal value in Galan (NEON).",
```

Comment:
ping

Reply:
This gas-price should be setup in the transaction.
It give guaranties that it covers operator fees and slippage.

Reply:
gas-price in tx can be lower than suggested_gas_price.

gas-price in tx cannot be less than min_acceptable_gas_price, if gas-price in tx is less than min_acceptable_gas_price, it is rejected by the Proxy.

if gas-price less than min_executable_gas_price, tx is accepted by the Proxy to mempool, but it waits for order of the execution till min_executable_gas_price go down and gas-price in tx becames enough for execution.